src/panels/config/automation/trigger/types/ha-automation-trigger-time_pattern.ts

```diff
@@ -19,37 +17,31 @@ export class HaTimePatternTrigger extends LitElement implements TriggerElement {
   }
 
   protected render() {
-    const { hours, minutes, seconds } = this.trigger;
     return html`
-      <paper-input
-        .label=${this.hass.localize(
-          "ui.panel.config.automation.editor.triggers.type.time_pattern.hours"
-        )}
-        name="hours"
-        .value=${hours}
+      <ha-form
+        .hass=${this.hass}
+        .schema=${[
+          { name: "hours", selector: { text: {} } },
+          { name: "minutes", selector: { text: {} } },
+          { name: "seconds", selector: { text: {} } },
+        ]}
+        .data=${this.trigger}
+        .computeLabel=${this._computeLabelCallback}
```

Comment:
I don't think this works? You pass a function, so the `this` will point at something else.